index.html

```diff
@@ -203,8 +204,33 @@ <h2>Challenges</h2>
                                     </div>
                                 </div>
                             </section>
+							-->
+							
+							<section id="challenges" class="main special">
+                                <div class="spotlight">
+                                    <div class="content">
+                                        <header class="major">
+                                            <h2>Tracks</h2>
+                                        </header>
+										<h3><strong>Choose a track (or combine multiple!) to focus your idea upon.</strong></h3>
```

Comment:
The indentation here and at line 218 onwards is really messed up. Fix this and then we can merge.

Reply:
Well that's odd, something must have changed my indentation during upload.
I'll hop on fixing that up.